docs/schema.md

```diff
@@ -158,6 +158,7 @@ The defined database prefixes and their "home" databases are:
 | `RUSTSEC` | [The Rust crates vulnerability database](https://github.com/rustsec/advisory-db). Serving `<ID>` in the shared format at  `https://github.com/RustSec/advisory-db/blob/osv/crates/<ID>.json` |
 | `GSD` | The GSD database. Serving the shared format [here](https://github.com/cloudsecurityalliance/gsd-database). |
 | `GHSA` | The GitHub Security Advisory database. Serving the shared format [here](https://github.com/github/advisory-database). |
+| `LBSEC` | The LoopBack Advisory Database. Serving the shared format [here](https://github.com/loopbackio/security/tree/main/advisories). |
```

Comment:
The examples in the accompanying repo indicate these will use an `LBSA` prefix, not `LBSEC` - is this still in flux?

Reply:
`LBSA-` was used interim when we were initially setting up the repo but we've since switched over to `LBSEC-`, which would be reflected in a PR that we're planning to land in the upcoming week or so.

Reply:
@achrinza thanks for clarifying!